src/com/facebook/buck/jvm/java/AccumulateClassNamesStep.java

```diff
@@ -163,10 +159,11 @@ public InputStream openStream() throws IOException {
     Map<String, HashCode> classNames = new HashMap<>();
 
     for (String line : lines) {
-      List<String> parts = CLASS_NAME_AND_HASH_SPLITTER.splitToList(line);
-      Preconditions.checkState(parts.size() == 2);
-      String key = parts.get(0);
-      HashCode value = HashCode.fromString(parts.get(1));
+      int index = line.lastIndexOf(CLASS_NAME_HASH_CODE_SEPARATOR);
+      Preconditions.checkArgument(
+          index > 0, String.format("Class names and hashcodes malformed: %1$s", line));
```

Comment:
Can you please use the form of checkArgument that takes a format string + arguments? We don't want to allocate a string each time that we are in this loop for the ideally non-occurring case that index < 0

https://guava.dev/releases/21.0/api/docs/com/google/common/base/Preconditions.html#checkArgument-boolean-java.lang.String-java.lang.Object...-